suite/reorg/direcct_request.go

```diff
@@ -1,67 +1,105 @@
-package smoke
+package reorg
```

Comment:
Why not move this to core with the others?

Reply:
The same as for soak tests.